zap/src/config.rs

```diff
@@ -432,8 +432,17 @@ impl Range {
 		}
 	}
 
-	pub fn numty(&self) -> Option<NumTy> {
-		Some(NumTy::from_f64(self.min.unwrap_or(0.0), self.max?))
+	pub fn numty(&self) -> Option<(NumTy, f64)> {
+		let min = self.min.unwrap_or(0.0);
+		let max = self.max?;
```

Comment:
When `self.max` isn't defined, why don't we return `(NumTy::U16, 0)` here instead of this function returning a `None`?